[Merged by Bors] - Feat: siren faq update (#4685)
sigp/lighthouse: book/src/ui-faqs.md

## 2. Where can I find my API token?
The required Api token may be found in the default data directory of the validator client. For more information please refer to the lighthouse ui configuration [`api token section`](./api-vc-auth-header.md).

## 3. How do I fix the Node Network Errors?
If you receive a red notification with a BEACON or VALIDATOR NODE NETWORK ERROR you can refer to the lighthouse ui configuration and [`connecting to clients section`](./ui-configuration.md#connecting-to-the-clients).

## 4. How do I connect Siren to Lighthouse from a different computer on the same network?
The most effective approach to enable access for a local network computer to Lighthouse's HTTP API ports is by configuring the `--http-address` to match the local LAN IP of the system running the beacon node and validator client. For instance, if the said node operates at `192.168.0.200`, this IP can be specified using the `--http-address` parameter as `--http-address 192.168.0.200`.
Subsequently, by designating the host as `192.168.0.200`, you can seamlessly connect Siren to this specific beacon node and validator client pair from any computer situated within the same network.

> ## 5. How do I connect Siren to Lighthouse via a ssh tunnel?

Reviewer: I would suggest putting this point after 6. I think 6. Is more useful and then it can reference this one IF people want to go down the ssh tunnel path

> If you would like to access Siren beyond the local computer, we recommend using an SSH tunnel. This requires a tunnel for 3 ports: `80` (assuming the port is unchanged as per the [installation guide](./ui-installation.md#docker-recommended), `5052` (for beacon node) and `5062` (for validator client). You can use the command below to perform SSH tunneling:

Reviewer: ```suggestion
If you would like to access Siren beyond the local network (i.e across the internet), we recommend using an SSH tunnel. This requires a tunnel for 3 ports: `80` (assuming the port is unchanged as per the [installation guide](./ui-installation.md#docker-recommended), `5052` (for beacon node) and `5062` (for validator client). You can use the command below to perform SSH tunneling:
```

```bash

ssh -N -L 80:127.0.0.1:80 -L 5052:127.0.0.1:5052 -L 5062:127.0.0.1:5062 username@local_ip

```

Where `username` is the username of the server and `local_ip` is the local IP address of the server. Note that with the `-N` option in an SSH session, you will not be able to execute commands in the CLI to avoid confusion with ordinary shell sessions. The connection will appear to be "hung" upon a successful connection, but that is normal. Once you have successfully connected to the server via SSH tunneling, you should be able to access Siren by entering `localhost` in a web browser.

You can also access Siren using the app downloaded in the [Siren release page](https://github.com/sigp/siren/releases). To access Siren beyond the local computer, you can use SSH tunneling for ports `5052` and `5062` using the command:

```bash

ssh -N -L 5052:127.0.0.1:5052 -L 5062:127.0.0.1:5062 username@local_ip

```

## 6. How can I use Siren to monitor my validators remotely when I am not at home?

There are two primary methods to access your Beacon Node and Validator Client remotely: setting up a VPN or utilizing SSH-reverse tunneling.

> Most contemporary home routers provide options for VPN access in various ways. A VPN permits a remote computer to establish a connection with internal computers within a home network. With a VPN configuration in place, connecting to the VPN enables you to treat your computer as an integral part of the local network. The connection process involves following the setup steps for connecting via another machine on the same network on the lighthouse ui configuration and [`connecting to clients section`](./ui-configuration.md#connecting-to-the-clients).

Reviewer: ```suggestion
Most contemporary home routers provide options for VPN access in various ways. A VPN permits a remote computer to establish a connection with internal computers within a home network. With a VPN configuration in place, connecting to the VPN enables you to treat your computer as if it is part of your local home network. The connection process involves following the setup steps for connecting via another machine on the same network on the Siren configuration page and [`connecting to clients section`](./ui-configuration.md#connecting-to-the-clients).
```

> In the absence of a VPN, an alternative approach involves utilizing a reverse SSH tunnel. To achieve this, you need remote SSH access to the computer hosting the Beacon Node and Validator Client pair (which necessitates a port forward in your router). In this context, while it is not obligatory to set a `--http-address` flag on the Beacon Node and Validator Client, you can configure an SSH tunnel to the local ports on the node and establish a connection through the tunnel. For instructions on setting up an SSH tunnel, refer to [`Connecting Siren via SSH tunnel`](./ui-faqs.html#5-how-do-i-connect-siren-to-lighthouse-via-a-ssh-tunnel) for detailed guidance.

Reviewer 2: I think the `--http-address` (along with `--unencrypted-http-transport`) flag is always required for validator client, as there is no default value (unlike the beacon node, which has a default value `127.0.0.1`)

Reviewer 3: > I think the `--http-address` (along with `--unencrypted-http-transport`) flag is always required for validator client, as there is no default value (unlike the beacon node, which has a default value `127.0.0.1`)

I am not sure about this. The last time I tested Siren remotely via ssh tunneling, `http-address` is not needed and I can still connect. About `--unencrypted-http-transport`, I don't know if that is required as well (in case `--http-address` is needed`). When I tested the validator manager, I commented on this and it was removed from the doc:
https://github.com/sigp/lighthouse/pull/3502#discussion_r1267952787

Reviewer 2: ah yeah you're right! I've just checked [the code](https://github.com/sigp/lighthouse/blob/1373dcf076bd031e1c09119281c5f84b789974c2/validator_client/src/http_api/mod.rs#L104-L105) and can confirm `--http-address` is not required! but `unencrypted-http-transport` would be required if `--http-address` is present.

Author: okay so ill mention that `unencrypted-http-transport` is necessary when `--http-address` is present

> In the absence of a VPN, an alternative approach involves utilizing a reverse SSH tunnel. To achieve this, you need remote SSH access to the computer hosting the Beacon Node and Validator Client pair (which necessitates a port forward in your router). In this context, while it is not obligatory to set a `--http-address` flag on the Beacon Node and Validator Client, you can configure an SSH tunnel to the local ports on the node and establish a connection through the tunnel. For instructions on setting up an SSH tunnel, refer to [`Connecting Siren via SSH tunnel`](./ui-faqs.html#5-how-do-i-connect-siren-to-lighthouse-via-a-ssh-tunnel) for detailed guidance.

Reviewer: ```suggestion
In the absence of a VPN, an alternative approach involves utilizing an SSH tunnel. To achieve this, you need remote SSH access to the computer hosting the Beacon Node and Validator Client pair (which necessitates a port forward in your router). In this context, while it is not obligatory to set a `--http-address` flag on the Beacon Node and Validator Client, you can configure an SSH tunnel to the local ports on the node and establish a connection through the tunnel. For instructions on setting up an SSH tunnel, refer to [`Connecting Siren via SSH tunnel`](./ui-faqs.html#5-how-do-i-connect-siren-to-lighthouse-via-a-ssh-tunnel) for detailed guidance.
```
